lh-test/src/main/java/io/littlehorse/test/LHExtension.java

```diff
@@ -35,10 +39,20 @@ public void postProcessTestInstance(Object testInstance, ExtensionContext contex
         try {
         List<LHTaskWorker> workers = testContext.discoverTaskWorkers(testInstance);
         for (LHTaskWorker worker : workers) {
-            store.put(worker.getTaskDefName(), worker);
-                worker.registerTaskDef(true);
-                worker.start();
+            if(store.get(worker.getTaskDefName()) != null){
```

Comment:
Basically there's a store of all `TaskDef`'s we have, and we only create each one once?

I like it. The thing is maybe we should put some documentation somewhere that says that if two different `LHTest`s use the same `LHTaskMethod` annotation, then they need to have matching method signatures.

Reply:
yeah, i'll be adding documentation for this